site/templates/base.html

```diff
@@ -7,7 +7,7 @@
         <title>
             {% block header %}{% endblock %}
         {{ SITE_TITLE }}</title>
-        <link rel="stylesheet" href="/static/tailwind.css" />
+        <link rel="stylesheet" href="/static/style.css" />
```

Comment:
I changed the name here just to avoid some confusion, but I have no strong attachments to this change.